public/app/features/explore/Graph/GraphContainer.tsx

```diff
@@ -76,19 +77,26 @@ export const GraphContainer = ({
 
   return (
     <PanelChrome
-      title="Graph"
+      title={t('graph.container.title', 'Graph')}
       titleItems={[
         !showAllSeries && MAX_NUMBER_OF_TIME_SERIES < data.length && (
           <div key="disclaimer" className={styles.timeSeriesDisclaimer}>
             <span className={styles.warningMessage}>
               <Icon name="exclamation-triangle" aria-hidden="true" />
-              Showing only {MAX_NUMBER_OF_TIME_SERIES} series
+              <Trans i18nKey={'graph.container.show-only-series'} count={MAX_NUMBER_OF_TIME_SERIES}>
```

Comment:
I think because `MAX_NUMBER_OF_TIME_SERIES` will never be 1, both this and the other plural translations don't actually need to be plural translations, but maybe someone from @grafana/explore-squad can give their opinion on that?

Reply:
Awesome. Should I wait for the squad's message or make changes already @JoaoSilvaGrafana  ?

Reply:
Feel free to make the changes now, but I'd wait for the squad for approval afterwards 👍

Reply:
Yep, that's a good point. It should never be 1 so it doesn't need to be a plural 👍